modules/simulation/JsYetiSimulator/index.js

```diff
@@ -16,10 +14,9 @@ const STATE_F = 6;
 const Event_PowerOn = 8;
 const Event_PowerOff = 9;
 
-var module_id;
 let global_info;
 
-let active_errors = {
+const active_errors = {
```

Comment:
Is this really supposed to be const?

Reply:
Yeah that should be not a const. Thank you for seeing that